src/routes/__root.tsx

```diff
@@ -1,11 +1,9 @@
 import { createRootRoute, Outlet } from '@tanstack/react-router';
-import { TanStackRouterDevtools } from '@tanstack/router-devtools';
 
 export const Route = createRootRoute({
   component: () => (
     <>
       <Outlet />
-      <TanStackRouterDevtools />
```

Comment:
We could just hide it unless we're in dev mode/localhost?

Reply:
ill do that later